eventhubs/client/azure-eventhubs/src/main/java/com/azure/core/amqp/ExceptionHandler.java

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.amqp;
+
+import com.azure.core.amqp.exception.ErrorContext;
+import com.azure.core.implementation.logging.ServiceLogger;
+
+import java.util.Objects;
+
+/**
+ * Handles exceptions generated by AMQP connections, sessions, and/or links.
+ */
+public abstract class ExceptionHandler {
```

Comment:
What do you think about naming this AmqpExceptionHandler, as ExceptionHandler feels too generic to me.

Reply:
Sounds good. I'll update these names!